src/Microsoft.Extensions.Logging.Console/Internal/IAnsiSystemConsole.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Extensions.Logging.Console.Internal
+{
+    public interface IAnsiSystemConsole
```

Comment:
yeah I know...its more 'console' interfaces...I wanted to add tests for AnsiLogConsole as it has considerable amount of code...